src/SearchResults/index.js

```diff
@@ -694,6 +768,9 @@ function vanillaSortFn(order, data) {
  * might not be respected if you have facet values that are already refined.
  * @param {string} attribute attribute name
  * @param {object} opts configuration options.
+ * @param {boolean} [opts.facetOrdering=false]
+ * Force the use of facetOrdering from the result if a sortBy is present. If
+ * sortBy isn't present, facetOrdering will be used automatically.
```

Comment:
Then it's not `false` by default? (since it's computed based on `sortBy`)

```suggestion
 * @param {boolean} [opts.facetOrdering]
 * Force the use of facetOrdering from the result if a sortBy is present. If
 * sortBy isn't present, facetOrdering will be used automatically.
```